lib/expression.ts

```diff
@@ -68,12 +68,15 @@ class ExpressionTranspiler extends base.TranspilerBase {
 
       case ts.SyntaxKind.PropertyAccessExpression:
         var propAccess = <ts.PropertyAccessExpression>node;
-        this.visit(propAccess.expression);
         if (propAccess.name.text === 'stack' &&
             this.hasAncestor(propAccess, ts.SyntaxKind.CatchClause)) {
+          // Handle `e.stack` accesses in catch clauses by mangling to `e_stack`.
+          // FIXME: Use type checker/FacadeConverter to make sure this is actually Error.stack.
+          this.visit(propAccess.expression);
           this.emitNoSpace('_stack');
```

Comment:
- what about adding a test here ?
- could this be solved by adding a transformer that visits the catch clause ? We should be able to determine if `.stack` is accessed on the exception ?